wiki/Contests/PHFC/2024/en.md

```diff
@@ -0,0 +1,67 @@
+---
+tags:
+  - phfc
+  - phfc 2024
+---
+
+# osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"
+
+![](img/logo.png)
+
+The **osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest made for the Philippine osu! community featuring its four mascots. It is themed after the Christmas season.
```

Comment:
```suggestion
The **osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest run by the Philippine osu! community featuring its four mascots. It is themed after the Christmas season.
```

Reply:
I believe the issue here was the use of "made". Removed that word and shuffled the sentencse a little bit. I believe the current wording reads nicer as well.